ss/pebbledb/db.go

```diff
@@ -222,6 +222,11 @@ func (db *Database) Get(storeKey string, targetVersion int64, key []byte) ([]byt
 }
 
 func (db *Database) ApplyChangeset(version int64, cs *proto.NamedChangeSet) error {
+	// Check if version is 0 and change it to 1
```

Comment:
I would add a specific note about genesis, i.e. "why" we're doing this.

Reply:
Updated